nettacker/locale/en.yaml

```diff
@@ -117,4 +117,9 @@ username_list: username(s) list, separate with ","
 verbose_mode: verbose mode level (0-5) (default 0)
 wrong_hardware_usage: "You must select one of these profiles for hardware usage. (low, normal, high, maximum)"
 invalid_scan_id: your scan id is not valid!
-
+compare_scans: compare current scan to old scans using the unique scan_id
```

Comment:
@Captain-T2004 can you add these messages to another language file e.g. Hindi hi.yaml?

Reply:
Yeah, sure